core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpAuthConnectionContext.java

```diff
@@ -86,7 +86,14 @@ IOContextResult handleIO(NetworkIOJob netIoJob) {
         if (authenticated) {
             return super.handleIO(netIoJob);
         }
-        return handleAuth();
+        IOContextResult result = handleAuth();
+        if (authenticated && recvBufPos > recvBufStart) {
+            // authentication is completed and there are still bytes remaining in the buffer
```

Comment:
I think `recvBufPos > recvBufStart` does not mean bytes remain in the buffer. I don't see `recvBufPos` set in `findLineEnd()`.
I think this condition will be true always after signed challenge is received.

Reply:
> I don't see `recvBufPos` set in `findLineEnd()`.

It's set in `waitForResponse()` -> `compactBuffer()`